### ISSUE TITLE
Enable master page as reposiory page

### ISSUE THREAD

**Issue by e200** [UseKamba/kamba_checkout_web#3]

**Comment by amarildolucas** [comment]
Não entendi muito bem "Enable master branch as repository page"?

**Comment by e200** [comment]
Sim, preciso que habilites o repositório master como página deste repositório para que eu possa gerar uma página estática contendo código javascript de testes unitários, ex: https://usekamba.github.io/CheckoutJS/tests/checkout.test.html e assim todo mundo passa a poder ver os testes rodar por aqui mesmo apenas acessando este endereço.

**Comment by e200** [comment]
Tal como fiz aqui: https://e200.github.io/anotherJS/tests/test.html

**Comment by amarildolucas** [comment]
Ah! Então o melhor é termos a documentação diretamente no “readme” mesmo. E a outra documentação é gerada através da API usando Slate, e ainda tem a documentação no site. Ou seja tenderá a confundir sobre qual a documentação padrão e fora as atualizações em diversos lugares diferentes de forma manual. Nós usaremos Slate para a documentação padrão da referência geral e “readme” para trechos de código e sdk separadamente. Ou pelo menos podemos simplesmente começar com a biblioteca e o readme focado em si. E adicionarmos coisas a medida que evoluímos. É preferível adicionarmos ferramenta em relação a necessidade ao invés de monta uma Stack pré-definida.

Mas podes sempre criar testes, é sempre melhor representado algo implementado do que falarmos de seria ótimo ou não. Mas como a equipa é pequena o ideal é automatizarmos algumas tasks.

**Comment by e200** [comment]
Acho que não percebeste.

Existe uma biblioteca chamada QUnit, quando desenvolvo em JS uso-a para fazer testes unitários, por ser JS é possível eu disponibilizar os testes para todos e não deixá-los apenas nas nossas máquinas.

Imagina você acessar um link e saber se a tua biblioteca está a funcionar correctamente... Tu abres o link, o link roda os testes todos e te mostra os resultados.

**Comment by e200** [comment]
A ideia não é escrever a documentação em HTML

**Comment by amarildolucas** [comment]
Ata! Achei que fosse algo haver com a documentação do repositório. Então, para a Stack JS ou ferramentas a usar não há a necessidade de abrirmos uma **issue** enquanto não termos mais colaboradores. A não ser que seja algo que minha contribuição seja relevante. Depois da primeira Build teremos mais alguns colaboradores, já há pessoas interessadas, mas não faz sentido adicionar ou permitir ninguém tão cedo. Memos porque as PRs todas devem ser por algo já usado, testado. :)
